### ISSUE TITLE
Trouble w/ Reproducing 76.1 MOTA @2DMOT15

### ISSUE THREAD

**Issue by imchwan** [ifzhang/FairMOT#171]
Dear @ifzhang ,

First of all, thank you for sharing your great work!

I'm planning to reproduce 2DMOT15 result. (76.1 MOTA using the DLA-34 baseline model)

I have trained the model using the script you provided : "sh experiments/all_dla34.sh". (using default parameters)
I've finished training and got final model : all_dla34/model_30.pth 
Using above trained model, I had **74.3%** MOTA @2DMOT15(val).
(scripts I used : `python track.py mot --load_model ../models/model_30.pth --conf_thres 0.6`)

FYI, I had **76.3%** MOTA using all_dla.pth you provided.
(scripts I used : `python track.py mot --load_model ../models/all_dla34.pth --conf_thres 0.6`)

You mentioned about finetuning using MOT15 DB,
do I need to finetune w/ MOT15(train) to get 76.1 MOTA?
Or is there other method(more training epoch?,etc.) to achieve 76.1 MOTA?

Thank you in advance!

**Comment by ifzhang** [comment]
Because we use part of the MOT15 train dataset as our validation set, so we don't finetune it on MOT15 train dataset. You can try to modify --conf_thres from 0.55-0.65. Do you use the coco_ctdet model as your pretrained model? I think after training 30 epochs it is easy to reach 76+ MOTA on the validation set. You can also test the MOTA using 20,25,27 epochs.

**Comment by imchwan** [comment]
Thank you for the quick response.

I've used `sh experiments/all_dla34.sh`, so Yes, I did used coco_ctdet model as my pretrained model.
all_dla34.sh : `python train.py mot --exp_id all_dla34 --gpus 0,1,2,3 --batch_size 12 --load_model '../models/ctdet_coco_dla_2x.pth'`

I've tested w/ 20,25,27 epochs, and the results were ..
20 epoch : 73.8% MOTA
25 epoch : 74.6% MOTA
27 epoch : 74.6% MOTA
30 epoch : 74.3% MOTA
also couldn't achieve 76.1. :( 

If you didn't finetune w/ MOT15 train dataset, I think it is all about hyper-parameter.
With all_dla34.pth(your model) and conf_thes=0.6, I got 76+ MOTA, so I really wanna achieve this with conf_thes=0.6.
(I will also check w/ conf_thres = 0.55-0.65)

Maybe what I really want is to reproduce all_dla34.pth.
Can you please give me any other guidance?

**Comment by ifzhang** [comment]
You can try batchsize 16 or 24 because you have 4 cards and accordingly adjust the learning rate.

**Comment by imchwan** [comment]
Thanks. I will have a look and update the status.
